launcher/teeserver/tee_server.go

```diff
@@ -204,3 +205,21 @@ func (s *TeeServer) Shutdown(ctx context.Context) error {
 	}
 	return nil
 }
+
+func (a *attestHandler) handleAttestError(w http.ResponseWriter, err error, message string) {
+	st, ok := status.FromError(err)
+	if ok {
+		switch st.Code() {
```

Comment:
Should this be more flexible than only Bad request or Internal? See go/http-canonical-mapping